src/utils/organization/add-organization-members.js

```diff
@@ -28,15 +28,15 @@ async function distributeUsersByExist(user) {
   }
 }
 
-async function addMember({ password, ...member }) {
-  const { organizationId, audience, pipe, membersKey } = this;
+async function addMember(member) {
+  const { organizationId, audience, pipe, membersKey, inviteAccepted } = this;
 
   const memberKey = redisKey(organizationId, ORGANIZATIONS_MEMBERS, member.id);
   const memberOrganizations = redisKey(member.id, USERS_METADATA, audience);
 
   member.username = member.email;
   member.invited = Date.now();
-  member.accepted = password ? Date.now() : null;
```

Comment:
logic is being changed here, how is the back-compatibility here?

Reply:
I suppose the logic here is not quite correct.

In the **distributeUsersByExist** [getUserId](https://github.com/sky3d/ms-users/blob/master/src/utils/organization/add-organization-members.js#L24) returns only `id` without password for existing users.
In the **registerOrganizationMember** method returns [the user with password](https://github.com/sky3d/ms-users/blob/master/src/utils/organization/register-organization-members.js#L64)

Then the existing users combines with new ones
`const organizationMembers = registeredMembers.concat(createdMembers);`
Some items in the merged list have a password but some do not. In the addMember method accepted-logic depends on the password. 

As a result `accepted` flag will never assigned for existing users. Password condition is not valid in this case and I pass the `{ inviteAccepted: true }` flag directly to determine the case when user invitation accepted .